include/nccl_ofi_tuner.h

```diff
@@ -5,6 +5,7 @@
 #ifndef NCCL_OFI_TUNER_H_
 #define NCCL_OFI_TUNER_H_
 
+#include "nccl_ofi.h"
```

Comment:
you shouldn't need this

Reply:
moved it into the second config.h file as requested, thanks